client/app/telecom/pack/resiliation/pack-resiliation.html

```diff
@@ -47,7 +47,8 @@ <h1 data-translate="pack_resiliation_title"></h1>
                             <p class="form-control-static">
                                 <strong class="text-price"
                                         data-ng-if="!PackResiliation.computingPrice"
-                                        data-ng-bind="PackResiliation.resiliationTerms.data.due.text"></strong>
+                                        data-translate="pack_resiliation_excluding_tax"
```

Comment:
According to the translation value (ref: https://github.com/ovh-ux/ovh-manager-telecom/blob/7ddd35c7ccde0f55dd217852949c5ef83e9c85a9/client/app/telecom/pack/resiliation/translations/Messages_fr_FR.xml#L43) does `{{currency}}` is needed?

Reply:
It may be needed if you look at line 234 of client/app/telecom/pack/resiliation/pack-resiliation.html file